doc/rbd/iscsi-monitoring.rst

```diff
@@ -0,0 +1,103 @@
+-----------------------------
+Monitoring the iSCSI gateways
+-----------------------------
+
+Ceph provides an additional tool for iSCSI gateway environments
+to monitor performance of exported RADOS Block Device (RBD) images.
+
+The ``gwtop`` tool is a ``top``-like tool that displays aggregated
+performance metrics of RBD images that are exported to clients over
+iSCSI. The metrics are sourced from a Performance Metrics Domain Agent
+(PMDA). Information from the Linux-IO target (LIO) PMDA is used to list
+each exported RBD image with the connected client and its associated I/O
+metrics.
+
+**Requirements:**
+
+-  A running Ceph iSCSI gateway
+
+**Installing:**
+
+1. As ``root``, install the ``ceph-iscsi-tools`` package on each iSCSI
```

Comment:
@ritz303 following patch can re-number the items correctly. i tested it locally.

```diff
diff --git a/doc/rbd/iscsi-monitoring.rst b/doc/rbd/iscsi-monitoring.rst
index b3de0094db..5cdd3d5133 100644
--- a/doc/rbd/iscsi-monitoring.rst
+++ b/doc/rbd/iscsi-monitoring.rst
@@ -18,37 +18,27 @@ metrics.

 **Installing:**

-1. As ``root``, install the ``ceph-iscsi-tools`` package on each iSCSI
-   gateway node:
-
-   ::
+#. As ``root``, install the ``ceph-iscsi-tools`` package on each iSCSI
+   gateway node: ::

        # yum install ceph-iscsi-tools

-2. As ``root``, install the performance co-pilot package on each iSCSI
-   gateway node:
-
-   ::
+#. As ``root``, install the performance co-pilot package on each iSCSI
+   gateway node: ::

        # yum install pcp

-3. As ``root``, install the LIO PMDA package on each iSCSI gateway node:
-
-   ::
+#. As ``root``, install the LIO PMDA package on each iSCSI gateway node: ::

        # yum install pcp-pmda-lio

-4. As ``root``, enable and start the performance co-pilot service on
-   each iSCSI gateway node:
-
-   ::
+#. As ``root``, enable and start the performance co-pilot service on
+   each iSCSI gateway node: ::

        # systemctl enable pmcd
        # systemctl start pmcd

-5. As ``root``, register the ``pcp-pmda-lio`` agent:
-
-   ::
+#. As ``root``, register the ``pcp-pmda-lio`` agent: ::

        cd /var/lib/pcp/pmdas/lio
        ./Install
```